test/generator/src/main/java/JavaScriptTestDataGenerator.java

```diff
@@ -74,10 +76,16 @@ private static Map<String, StringBuilder> getCqlSnippets(Path file) throws IOExc
     }
 
     public static void loadModelInfo(File modelInfoXML, ModelManager modelManager) {
-        final ModelInfo modelInfo = JAXB.unmarshal(modelInfoXML, ModelInfo.class);
-        final VersionedIdentifier modelId = new VersionedIdentifier().withId(modelInfo.getName()).withVersion(modelInfo.getVersion());
-        final ModelInfoProvider modelProvider = (VersionedIdentifier modelIdentifier) -> modelInfo;
-        modelManager.getModelInfoLoader().registerModelInfoProvider(modelProvider);
+        try {
+            final ModelInfo modelInfo = ModelInfoReaderFactory.getReader("application/xml").read(modelInfoXML);
+            final ModelIdentifier modelId = new ModelIdentifier().withId(modelInfo.getName()).withVersion(modelInfo.getVersion());
+            final ModelInfoProvider modelProvider = (ModelIdentifier modelIdentifier) -> modelInfo;
+            final ModelInfoLoader modelInfoLoader = new ModelInfoLoader();
```

Comment:
Whoops.  I don't need this line.  It's left over from some debugging.  I'll remove it.